src/debugging/netcore/VsDbgHelper.ts

```diff
@@ -34,7 +34,7 @@ const acquisition: { url: string, scriptPath: string, getShellCommand(runtime: V
             url: 'https://aka.ms/getvsdbgsh',
             scriptPath: path.join(vsDbgInstallBasePath, 'getvsdbg.sh'),
             getShellCommand: (runtime: VsDbgRuntime, version: VsDbgVersion) => {
-                return `${acquisition.scriptPath} -v ${version} -r ${runtime} -l "${getInstallDirectory(runtime, version)}"`;
+                return `chmod +x ${acquisition.scriptPath} && ${acquisition.scriptPath} -v ${version} -r ${runtime} -l "${getInstallDirectory(runtime, version)}"`;

```

Comment:
Should `${acquisition.scriptPath}` be quoted (like we do for the installation directory)?

Reply:
Yes, good catch.